module-sys/Service/Service.cpp

```diff
@@ -280,9 +263,6 @@ namespace sys
     {
         auto ret = ReturnCodes::Success;
         switch (message->systemMessageType) {
-        case SystemMessageType::Ping:
```

Comment:
what was this message type used for ? or it was not used at all?

Reply:
it's a legacy Ping message :D it was used for some code proofing when the bus was added, then it was abandoned.
Now it's not used at all - this is why I could remove it :)